src/main.rs

```diff
@@ -96,6 +105,41 @@ async fn main() -> std::io::Result<()> {
         .new_game(GameOptions {}, Some("test".to_string()))
         .unwrap();
 
+    let reaper_manager_ref = game_manager_wrapper.clone();
+
+    // Game Reaper thread
+    thread::spawn(move || loop {
```

Comment:
It'd be nice if this whole thing was its own function probably, in a different file like `periodic.rs`.

Reply:
seeemsss... doable... > has not created a new file but it can't be that hard

Reply:
Ya just a fresh file and add it to `lib.rs`.

Reply:
lib.rs that was the trick here (y)